site/content/en/docs/Reference/gameserver.md

```diff
@@ -245,10 +246,11 @@ The `spec` field is the actual GameServer specification and it is composed as fo
 - `container` is the name of container running the GameServer in case you have more than one container defined in the [pod](https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/). If you do,  this is a mandatory field. For instance this is useful if you want to run a sidecar to ship logs.
 - `ports` are an array of ports that can be exposed as direct connections to the game server container
   - `name` is an optional descriptive name for a port
-  - `portPolicy` has three options:
+  - `portPolicy` has four options:
```

Comment:
These changes will need to live behind a `feature` shortcode please (our docs publish immediately on merge).

See: https://agones.dev/site/docs/contribute/ for details.